crates/voicevox_core/src/result_code.rs

```diff
@@ -79,5 +89,10 @@ pub const fn error_result_to_message(result_code: VoicevoxResultCode) -> &'stati
             "すでに読み込まれているModelを読み込もうとしました\0"
         }
         VOICEVOX_UNLOADED_MODEL_ERROR => "Modelが読み込まれていません\0",
+        VOICEVOX_USER_DICT_READ_ERROR => "辞書を読み込めませんでした\0",
+        VOICEVOX_USER_DICT_WRITE_ERROR => "辞書を書き込めませんでした\0",
+        VOICEVOX_WORD_NOT_FOUND_ERROR => "辞書に単語が見つかりませんでした\0",
+        VOICEVOX_USER_DICT_LOAD_ERROR => "辞書をOpenJTalkに読み込ませることができませんでした\0",
+        VOICEVOX_INVALID_WORD_ERROR => "単語のバリデーションに失敗しました\0",
```

Comment:
エラーしっかりしていていいですね！！！
システム辞書と間違えそう、単語だと何かわからなそうかなとか思いました！
こんな感じでどうでしょう。

```suggestion
        VOICEVOX_USER_DICT_READ_ERROR => "ユーザー辞書を読み込めませんでした\0",
        VOICEVOX_USER_DICT_WRITE_ERROR => "ユーザー辞書を書き込めませんでした\0",
        VOICEVOX_USER_DICT_WORD_NOT_FOUND_ERROR => "ユーザー辞書に単語が見つかりませんでした\0",
        VOICEVOX_USER_DICT_LOAD_ERROR => "ユーザー辞書をOpenJTalkに読み込ませることができませんでした\0",
        VOICEVOX_USER_DICT_INVALID_WORD_ERROR => "UserDictWordのバリデーションに失敗しました\0",
```

Reply:
なるほど。学校終わったら置き換えます